src/components/token/TokenBalances.js

```diff
@@ -226,18 +231,33 @@ class TokenBalances extends React.Component {
     });
   }
 
+  fetchHTRTransactionCount = async () => {
+    const tokenApiRequest = await tokensApi.getToken('00');
```

Comment:
We should get HTR uid from the lib constant

Reply:
Thanks, done in 7063b34535707b9b049b7aa921bedaec618e73e4